buzzmobile/tests/unit/process/test_bearing.py

```diff
@@ -16,20 +16,16 @@ def test_distance(self):
         assert np.isclose(calculate_directions.get_distance(fix1, fix2), 2517000, rtol=.01)
 
 
-@with_roscore
-class TestBearingNode(unittest.TestCase):
-
+class TestBearingNode(RosTest):
     @with_launch_file('buzzmobile', 'test_params.launch')
     @launch_node('buzzmobile', 'bearing.py')
     def test_bearing_node(self):
-        with mock_pub('/fix', NavSatFix, queue_size=None) as fix_node:
+        with mock_pub('/fix', NavSatFix, queue_size=0) as fix_node:
```

Comment:
No, `None` is technically bad, but its not a big issue, so I just swapped it bc y not.